compute/test/createComputeHyperdisk.test.js

```diff
@@ -30,24 +29,13 @@ describe('Create compute hyperdisk', async () => {
   const prefix = 'hyperdisk-name-941ad2d';
   const diskName = `${prefix}${Math.floor(Math.random() * 1000 + 1)}`;
   const zone = 'europe-central2-b';
-  const disksClient = new DisksClient();
-  let projectId;
 
-  before(async () => {
-    projectId = await disksClient.getProjectId();
+  after(async () => {
     // Cleanup resources
```

Comment:
In this PR I also moved cleanup methods to after() to prevent keeping not used resources in our project after running the tests.